packages/pwa-kit-create-app/assets/bootstrap/js/config/sites.js.hbs

```diff
@@ -10,6 +10,59 @@
 module.exports = [
     {
         id: '{{answers.project.commerce.siteId}}',
+        {{#ifEquals answers.project.commerce.siteId "RefArchGlobal" }}
```

Comment:
Similar to the previous comment, this condition would apply to more people/cases than only for our demo site. Is this intentional?

Reply:
This will only apply if the generator is specifically given `RefArchGlobal` as the site id. Most generated projects will set their own site id so this shouldn't apply. But sure, I can change the condition here to only apply for the demo site instead.

Reply:
The updated condition will only apply this config for the demo site